oaknational/Oak-Web-Application
# [CUR-303] Feat/curric video explainer

## Description

- Updated static text for video description to be pulled from Sanity 

## How to test

1. Go to https://deploy-preview-2003--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/overview
2. Scroll down to the video
3. You should see the correct video description pulled from Sanity 

## Screenshots

How it used to look (delete if n/a):
<img width="635" alt="Screenshot 2023-10-11 at 11 05 49" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/afa4fa78-554f-41b6-8c31-846b19a5e1b5">

How it should now look:
<img width="1402" alt="Screenshot 2023-10-11 at 11 05 33" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/c46eec28-3c0f-467a-85b1-491c7ddf7035">


## Checklist
